src/index.ts

```diff
@@ -50,6 +51,7 @@ export const bindProxyAndYMap = <T>(p: Record<string, T>, y: Y.Map<T>) => {
       bindProxyAndYMap(pv, yv);
       p[k] = pv as unknown as T;
     } else if (
+      yv === null ||
```

Comment:
Is this covered by the test?

Reply:
I believe so. Without that line, I get the error `unsupported y type`

Reply:
Cool. Thanks for the confirmation.